### PR TITLE
Update CI files

### PR DESCRIPTION
**Pull request by pulpbot**
[noissue]

### CODE REVIEW

File: .github/workflows/scripts/script.sh [hunk 1]
```diff
@@ -145,7 +145,7 @@ if [[ "$TEST" == "upgrade" ]]; then
 
   # Running post upgrade tests
   git checkout ci_upgrade_test -- pulp_container/tests/
-  pytest -vv -r sx --color=yes --pyargs --capture=no --durations=0 pulp_container.tests.upgrade.post
+  pytest -v -r sx --color=yes --pyargs -capture=no pulp_container.tests.upgrade.post
```
Comment (mdellweg) [inline .github/workflows/scripts/script.sh:148]: Is this correct?
@fao89

Comment (fao89) [reply .github/workflows/scripts/script.sh:148]: yep, I forgot to change back, I added these to debug

Comment (mdellweg) [reply .github/workflows/scripts/script.sh:148]: And what about the double dash before "capture" ?

Comment (fao89) [reply .github/workflows/scripts/script.sh:148]: I didn't notice it, I need to change it at the plugin-template

Comment (fao89) [reply .github/workflows/scripts/script.sh:148]: done - https://github.com/pulp/plugin_template/pull/486

**Review by mdellweg — APPROVED** [review]

**Pull request merged**